emsdk_manifest.json

```diff
@@ -156,6 +156,56 @@
   },
 
 
+  {
+    "id": "node",
+    "version": "22.16.0",
+    "bitness": 32,
+    "arch": "x86",
+    "windows_url": "node-v22.16.0-win-x86.zip",
+    "activated_path": "%installation_dir%/bin",
+    "activated_path_skip": "node",
+    "activated_cfg": "NODE_JS='%installation_dir%/bin/node%.exe%'",
+    "activated_env": "EMSDK_NODE=%installation_dir%/bin/node%.exe%"
```

Comment:
At some point (just like with python) I do think we should remove the older version of these tools from this json file.  I don't see any point is keeping stuff we don't use around.   Maybe after 6 months or so?